VTEX - Orders API (PII version).json

```diff
@@ -5749,1561 +5843,319 @@
-                    "Missing",
-                    "StdDev",
-                    "Sum",
-                    "SumOfSquares",
-                    "Facets"
-                ],
-                "type": "object",
-                "properties": {
-                    "Count": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "Count."
-                    },
-                    "Max": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "Maximum."
-                    },
-                    "Mean": {
-                        "type": "number",
-                        "description": "Mean."
-                    },
-                    "Min": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "Minimum."
-                    },
-                    "Missing": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "Missing."
-                    },
-                    "StdDev": {
-                        "type": "number",
-                        "description": "Standard deviation."
-                    },
-                    "Sum": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "Sum."
-                    },
-                    "SumOfSquares": {
-                        "type": "integer",
-                        "format": "int64",
-                        "description": "Sum of squares."
-                    },
-                    "Facets": {
-                        "type": "string",
-                        "nullable": true,
-                        "description": "Facets."
-                    }
-                },
-                "example": {
-                    "Count": 84,
-                    "Max": 89,
-                    "Mean": 2.2261904761904763,
-                    "Min": 1,
-                    "Missing": 0,
-                    "StdDev": 9.660940100525016,
-                    "Sum": 187,
-                    "SumOfSquares": 8163,
-                    "Facets": {
-                        "origin": {
-                            "Fulfillment": {
-                                "Count": 68,
-                                "Max": 1,
-                                "Mean": 1,
-                                "Min": 1,
-                                "Missing": 0,
-                                "StdDev": 0,
-                                "Sum": 68,
-                                "SumOfSquares": 68,
-                                "Facets": null
+                                    "Mean": {
+                                        "type": "number",
+                                        "description": "Value mean."
```

Comment:
what does this mean?

Reply:
It was very difficult to test this field, so I kept the idea already documented and should improve it in the future.